toolkit/scss/forms/_option-select.scss

```diff
@@ -131,3 +131,15 @@
     }
   }
 }
+
+.govuk-option-select {
```

Comment:
Yeah, or split it into two files which this file then includes? 

I'll thumbs up either of the above.